org.kde.neochat.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "org.kde.neochat",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "neochat",
+    "rename-icon": "neochat",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "qtkeychain",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "cleanup": [
+                "mkspecs"
+            ],
+            "config-opts": [
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+                "-DLIB_INSTALL_DIR=/app/lib",
+                "-DBUILD_TRANSLATIONS=NO"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/frankosterfeld/qtkeychain/archive/v0.10.0.tar.gz",
+                    "sha256": "5f916cd97843de550467db32d2e10f218b904af5b21cfdfcc7c6425d7dfc3ec2"
+                }
+            ]
+        },
+        {
+            "name": "kquickimageeditor",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://invent.kde.org/libraries/kquickimageeditor.git",
+                    "branch": "master"
+                }
+            ]
+        },
+        {
+            "name": "libqmatrixclient",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "cleanup": [
+                "/bin/qmc-example",
+                "/include",
+                "/lib",
+                "/share"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/quotient-im/libQuotient.git",
+                    "tag": "0.6.2"
+                }
+            ]
+        },
+        {
+            "name": "cmark",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMARK_TESTS=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/commonmark/cmark/archive/0.29.0.tar.gz",
+                    "sha256": "2558ace3cbeff85610de3bda32858f722b359acdadf0c4691851865bb84924a6"
+                }
+            ]
+        },
+        {
+            "name": "neochat",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://invent.kde.org/network/neochat.git",
+                    "branch": "master"
```

Comment:
We now have a 1.0 stable branch, you can use it. The 1.0 tag will be created from this branch and the master branch now require libQuotient master.

Reply:
Can we get a tag for kquickimageeditor too? Thanks!